forest/daemon/tests/keystore_tests.rs

```diff
@@ -0,0 +1,41 @@
+// Copyright 2019-2023 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use std::path::PathBuf;
+
+use anyhow::Result;
+use assert_cmd::Command;
+use tempfile::TempDir;
+
+// https://github.com/ChainSafe/forest/issues/2499
+#[test]
```

Comment:
More cases should be implemented once https://github.com/ChainSafe/forest/issues/2528 is done.